tiledb/sm/rest/rest_profile.cc

```diff
@@ -178,6 +177,14 @@ std::string RestProfile::get_param(const std::string& param) const {
  * See issue [#727](https://github.com/nlohmann/json/issues/727) for details.
  */
 void RestProfile::save_to_file() {
+  // RestProfiles are immutable, so disallow overwrites.
+  if (saved_) {
+    throw RestProfileException(
+        "Failed to save \'" + name_ +
+        "\'; This profile has already been saved, "
+        "and must be explicitly removed in order to be replaced.");
```

Comment:
How does the user do this?
We can provide a hint here, but either way this will have be covered in Academy.  Do we have a list of doc impact anywhere?

Reply:
`RestProfile::remove_from_file` does that. Also a `bool overwrite` option was added to `RestProfile::save_to_file` to do that under the hood, if necessary.